rednotebook/gui/options.py

```diff
@@ -74,7 +74,7 @@ def get_value(self):
 class AutostartOption(TickOption):
     def __init__(self):
         self.autostart_file = os.path.expanduser(
-            "~/.config/autostart/rednotebook.desktop"
+            "~/.config/autostart/app.rednotebook.rednotebook.desktop"
```

Comment:
It would be nice to provide backwards compatibility: at the beginning of the function, if the old file exists create the new file and remove the old file. If you like, I can take care of this, however.

Reply:
If you wish to add the backward compatibility, all the better. Thanks.

Reply:
Alright, let's merge this then and I'll think about adding this later.